### DISCUSSION TITLE
Getting followed by a bunch of bots

### DISCUSSION THREAD

**Discussion by NetOpWibby** [community/community#52450] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

I'm subscribed to my Github feed in NetNewsWire, mostly to see what new repos people I follow are starring.

I noticed that I got several new followers (9) and all of their accounts:
- were created two weeks ago
- have no original repos (all forks)
- avatars have artifacts

I don't see how this affects my account's security, but I just find it weird.

**Comment by tynist** [comment]
I also noticed today that almost 7 new and all their accounts:

-  were created  yesterday 9th June 2023 and followed me yesterday
-   repos are 5
-   repo names are one or two alphanumeric characters and are all empty
-  their followers are from 50 and following 300
-  avatars have artifacts
-  followed by [CharlesDerek](https://github.com/Vanessaidss?tab=followers)

I hope my our account are safe too
